src/io/react-native/react-native-types.ts

```diff
@@ -1,32 +1,23 @@
 import * as React from 'react'
-import { HttpResponse } from 'serverlet'
 
 import { LogBackend } from '../../core/log/log'
 import {
   EdgeContext,
   EdgeContextOptions,
   EdgeFakeUser,
   EdgeFakeWorld,
-  EdgeFetchOptions,
   EdgeNativeIo
 } from '../../types/types'
 
-export interface ClientIo {
-  // Networking:
-  fetchCors: (url: string, opts: EdgeFetchOptions) => Promise<HttpResponse>
-}
-
 export interface WorkerApi {
   makeEdgeContext: (
-    clientIo: ClientIo,
```

Comment:
Is this going to be a breaking change? Or is this an internal type?

Reply:
Internal only.